packages/@o3r/test-helpers/src/prepare-test-env.ts

```diff
@@ -82,59 +91,68 @@ export async function prepareTestEnv(folderName: string, type: PrepareTestEnvTyp
   }
 
   // Remove existing app
-  if (existsSync(appFolderPath)) {
-    rmSync(appFolderPath, {recursive: true});
+  if (existsSync(workspacePath)) {
+    rmSync(workspacePath, {recursive: true});
   }
 
-  if (type === 'blank') {
-    createTestEnvironmentBlank({
-      appDirectory: folderName,
-      cwd: itTestsFolderPath,
-      globalFolderPath,
-      yarnVersion
-    });
-  } else {
-    // Create new base app if needed
-    const baseAppAngular = `base-app-angular${generateMonorepo ? '-monorepo' : ''}`;
-    await createTestEnvironmentAngular({
-      appName: 'test-app',
-      appDirectory: baseAppAngular,
-      cwd: itTestsFolderPath,
-      globalFolderPath,
-      yarnVersion,
-      angularVersion,
-      materialVersion,
-      replaceExisting: !process.env.CI,
-      generateMonorepo
-    });
+  const prepareFinalApp = (baseApp: string) => {
+    logger.debug?.(`Copying ${baseApp} to ${workspacePath}`);
+    const baseProjectPath = path.join(itTestsFolderPath, baseApp);
+    cpSync(baseProjectPath, workspacePath, { recursive: true, dereference: true, filter: (source) => !/node_modules/.test(source) });
+    packageManagerInstall(execAppOptions);
```

Comment:
```suggestion
if (existsSync(path.join(workspacePath, 'package.json'))) {
    packageManagerInstall(execAppOptions);
}
```
Otherwise it can fail if there is no `package.json`. Note that I am wondering why it doesn't fail in CI, whereas it fails on my machine when I only run `yarn nx test-int ama-sdk-create` from a clean state.